README.md

```diff
@@ -1,33 +1,80 @@
+**Delivery mindset**
+
+We believe in being pragmatic and moving fast, spending our energy on the right things, shipping code and then rapidly iterating on it. 
+
+_Behaviours_:
+- We simplify our work, focusing on getting it to the user as early as possible.
+- We practice [continuous delivery](https://en.wikipedia.org/wiki/Continuous_delivery)
+- We are trying to [make the change easy, before the easy change](https://twitter.com/KentBeck/status/250733358307500032).   
+- We aim to solve issues rapidly prioritising them based on impact they may have.
+
+**Inclusiveness & Respect** 
 
 We believe that our culture is important, and we all contribute to it regardless of our role or level.
 
->Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+_Behaviours_:
+ - We ensure to treat everyone we encounter with respect independently of their skillsets and levels of ability
+ - Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+ - We respect the opinion of others and are gracious in our interactions, whether meeting physically or digitally, whether friend or stranger, whether employee, customer or commenter.
+ - We come from different backgrounds, work in different roles, are different ages, and therefore have different levels of experience. All in all, we pull each other up, not bring each other down.
```

Comment:
someone reading this might feel it's a bit "etc" on the underrepresented group(s) they identify with, if we care we can specify more groups we would welcome in this point

Reply:
I think the mention of `background`, `roles` and `ages` is linked here to support the different `levels of experience`. I am not sure mentioned `underrepresented group(s)` will make this sentence particularly better, however I think we could another sentence.

Reply:
How about we rephrase this to take out the group characteristics and instead say "Our experiences and backgrounds differ"  to address John's point above?

Also I think we can call out more than an appreciation of the impact this has on level of experience here, how about:

"and therefore so do our perspectives, areas of expertise, and levels of experience"

So all together:

"Our experiences and backgrounds differ, and therefore so do our perspectives, areas of expertise, and levels of experience. All in all ..."

Reply:
@amyhughes Thanks, I like your wording and have now used it!